sdks/typescript/featurehub-repository/app/client_context.ts

```diff
@@ -250,3 +252,20 @@ export class ClientEvalFeatureContext extends BaseClientContext {
   }
 
 }
+
+export class LocalClientContext extends BaseClientContext {
+  constructor(environment: Environment) {
```

Comment:
I just realised what is going on here, this is the old client and we aren't maintaining it any longer and won't be releasing it. The "featurehub-repository" will be deprecated and removed when @IrinaSouth gives the go-ahead.

Reply:
should I revert changes made under `sdks/typescript/featurehub-repository/`?

Reply:
Yes please!